features/step_definitions/us152_search_for_deed_via_title_number_and_md_ref.rb

```diff
@@ -0,0 +1,24 @@
+When(/^I search the deed store$/) do
+  str_end = '/deed?md_ref=e-MD12344&title_number='
+  deed_end_point = Env.deed_api + str_end + @title_number.to_s
+  @response = HTTP.get(deed_end_point)
+
+  # Open the URL in browser to check information
+  head = 'http://10.10.10.10:9020/deed?md_ref=e-MD12344&title_number='
```

Comment:
Should be using a variable for the root address which should be loaded in when the app starts. there are multiple occurrences of this